docs/getting-started/try-online.md

```diff
@@ -6,9 +6,9 @@ This instance is subject to unpredictable load though, and work to stabilize its
 
 ## Alternatives
 
-### Public Heroku Instance
+### Public Linode Instance
 
-Try this link: https://pennchime.herokuapp.com/
+Try this link: https://penn-chime.phl.io/
```

Comment:
I think this is supposed to be an alternative to Linode on line 3. Ask @mdbecker if he would update the heroku instance to v1.1.2.